test-e2e/ipfs-gateway.js

```diff
@@ -73,6 +74,12 @@ await $(execaOptions)`${kuboBin} config --json Gateway.ExposeRoutingAPI false`
 await $(execaOptions)`${kuboBin} config --json Gateway.HTTPHeaders.Access-Control-Allow-Origin ${JSON.stringify(['*'])}`
 await $(execaOptions)`${kuboBin} config --json Gateway.HTTPHeaders.Access-Control-Allow-Methods  ${JSON.stringify(['GET', 'POST', 'PUT', 'OPTIONS'])}`
 
+for (const carFile of await glob([`${resolve(__dirname, 'fixtures', 'data')}/**/*.car`])) {
+  log('Loading *.car fixture %s', carFile)
+  const { stdout } = await $(execaOptions)`${kuboBin} dag import --pin-roots=false --offline ${carFile}`
+  stdout.split('\n').forEach(log)
+}
```

Comment:
not loading car fixtures in this ipfs-gateway has probably caused me more issues with testing than I should admit